app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+---
+
+We’re here to answer your questions and provide free advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+
+##Come to an event
+
+Our events are a great way to meet people and talk about any concerns or questions you have about becoming a teacher.
+
+###Train to Teach
+
+Train to Teach events are free to attend and you'll be able to:
```

Comment:
Do we want to say that these are run by the Department of Education? We did this in the new events design based on UR (@KeishaDfE can confirm) and also with a view to differentiating these events.

Reply:
@gemmadallmandfe I think adding 'run by Department for Education' to TTT events will provide reassurance and give them more weight